ethapi/committee_cert_json_test.go

```diff
@@ -0,0 +1,98 @@
+package ethapi
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/0xsoniclabs/sonic/scc"
+	"github.com/0xsoniclabs/sonic/scc/bls"
+	"github.com/0xsoniclabs/sonic/scc/cert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCommitteeCertificateJson_ToCommitteeCertificate_ConvertsToHealthyCertificate(t *testing.T) {
```

Comment:
As a follow-up task in a new PR: please add at least one end-to-end test case checking the produced JSON output formatting.

Since this is the public interface served to customers, this surfacing format is relevant and should be checked.

Reply:
https://github.com/0xsoniclabs/sonic-admin/issues/91